redisCacheModule.js

```diff
@@ -130,7 +143,7 @@ function redisCacheModule(config){
     }
     log(false, 'get() called:', {key: key});
     try {
-      var cacheKey = (cleanKey) ? cleanKey : key;
+      var cacheKey = (cleanKey) ? cleanKey : prefixKey(key);
```

Comment:
I'm scratching my head here. I don't remember what `cleanKey` is for and I can't find a single scenario where I use it. I've looked in `cache-service-redis`, `cache-service`, `superagent-cache`, and `superagent-cache-plugin`. I'm tempted to remove it. Do you see anything that indicates why it exists?

Reply:
I've scratched my head last night trying to figure it out too...

Reply:
Haha ok, let's get rid of it! You can remove it in this PR, or I can do it after merging. Doesn't matter to me.